homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
+  # Bronze
+  action-setup: todo
```

Comment:
Are we able to fetch all sound outputs?

Reply:
Currently there is no upstream support for that, not sure if possible, most TV APis are by people reverse engineering the TV behavior so not everything is supported